tools/run-tests.py

```diff
@@ -195,6 +195,49 @@ def run_jsc_stress(engine, arch):
         env={'PYTHONPATH': '.'})
 
 
+def _run_regression_tests(engine, files, is_fail, assert_js):
```

Comment:
nitpicking: according to the order of Popen params, `engine, assert_js, files, is_fail` would be more logical